libraries/botbuilder-azure-blobs/src/ignoreError.ts

```diff
@@ -37,6 +37,6 @@ export async function ignoreError<T>(promise: Promise<T>, ignore: IgnoreError):
 export function isStatusCodeError(...codes: number[]): IgnoreError {
     const ignoredCodes = new Set(codes);
     return function (err) {
-        return err instanceof RestError && ignoredCodes.has(err.statusCode);
+        return err instanceof RestError && err.statusCode != null && ignoredCodes.has(err.statusCode);
```

Comment:
Do we have a test for this behavior?

Reply:
This change was also inspired by strict mode, but yes it's covered via https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder-azure-blobs/tests/blobsStorage.test.js#L61.

Reply:
Great, thanks!